earthly/python/poetry.lock

```diff
@@ -0,0 +1,153 @@
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
Why do we need poetry files here ?
All example project or project on which you gonna test the builder should be inside `examples` dir

Reply:
Deleted the files in commit 4f22d7aea18d573a03bc3ca0b87efbc43add4da0